### PR TITLE
Upgrade to latest netty-tcnative

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

**Comment by nmittler** [comment]
@ejona86 PTAL

File: build.gradle [hunk 1]
```diff
@@ -113,6 +113,15 @@ subprojects {
         if (osdetector.classifier in ["linux-x86_64", "osx-x86_64", "windows-x86_64"]) {
             // The native code is only pre-compiled on certain platforms.
             tcnative_suffix = ":" + osdetector.classifier
+            // RedHat variants use a different soname for OpenSSL than other linux distributions
+            // (see http://netty.io/wiki/forked-tomcat-native.html). Netty-tcnative, however
+            // assumes RedHat as the default "linux", while calling out "_debian" as a special
+            // build for debian-based systems. This should be inverted in future releases of
+            // netty-tcnative, but in the meantime we invert the logic manually here so that
+            // "debian" is used as the default linux distribution.
+            if (osdetector.os == "linux" && !osdetector.release.isLike("rhel")) {
```
Comment (ejona86) [inline build.gradle]: s/rhel/fedora/

**Comment by ejona86** [comment]
@nmittler, LGTM, other than fedora vs rhel

**Comment by nmittler** [comment]
Cherry-picked as d59d6dc

**Pull request closed without merging**